services/web/server/src/simcore_service_webserver/projects/projects_utils.py

```diff
@@ -69,6 +72,11 @@ def _replace_uuids(node):
         )
         if "mode" in project_copy["ui"]:
             project_copy["ui"]["mode"] = project_copy["ui"]["mode"]
+    if clean_output_data:
+        FIELDS_TO_DELETE = ["outputs", "progress", "runHash"]
```

Comment:
MINOR: ``FIELDS_TO_DELETE`` is an inmutable group of constant strings. Use ``tuple`` instead of ``list`` and place it outside the function so it is created once to infer that message to in your code

Reply:
yes very nice